lib/data/interceptor/auth_interceptor.dart

```diff
@@ -26,66 +25,32 @@ class AuthInterceptor extends InterceptorsWrapper {
   /// Called if token has expired.
   final VoidCallback onTokenExpired;
 
-  /// Whether request requires authentication or not.
-  bool isAuthenticatedPath(RequestOptions options) {
-    return !HttpClient.nonAuthenticatedPaths.contains(options.path);
-  }
-
   @override
   void onRequest(RequestOptions options, RequestInterceptorHandler handler) {
     // Don't add Token into paths that don't require authentication.
-    if (isAuthenticatedPath(options) && authPreferences.hasValidAccessToken) {
+    if (authPreferences.hasValidAccessToken) {
       options.headers['Authorization'] =
           'Bearer ${authPreferences.accessToken}';
     }
     handler.next(options);
   }
 
-  _TokenRefreshStatus _tokenRefreshState = _TokenRefreshStatus.none;
-  DioError? _tokenRefreshDioError;
-
   @override
-  Future onError(DioError err, ErrorInterceptorHandler handler) async {
+  void onError(DioError err, ErrorInterceptorHandler handler) async {
     final requestOptions = err.requestOptions;
 
     // If it is not an authenticated path or it is
```

Comment:
Small change, but I think the comment here should no longer include `If it is not an authenticated path`, so it doesn't mislead.